src/python/pants/notes/2.17.x.md

```diff
@@ -1,5 +1,54 @@
 # 2.17.x Release Series
 
+## What's New
+
+### Overall
+
+The [`scie-pants` launcher binary](https://github.com/pantsbuild/scie-pants) is now the recommended way to run Pants.
+Use of a `./pants` shell script in a repository is deprecated. Among other benefits, the `scie-pants` launcher is able
+to automatically download and maintain a Python distribution for running Pants. It also allows you to run `pants`
+from any directory within a repository and not just the build root. See the [Pants  installation
+instructions](https://www.pantsbuild.org/docs/installation) for additional information on how to migrate to `scie-pants.`
```

Comment:
Because we now include the native client in the wheel, `2.17.x` runs using `pantsd` are also about 4.25x faster: https://github.com/pantsbuild/pants/pull/18957